test/GPv2Settlement.test.ts

```diff
@@ -476,7 +477,7 @@ describe("GPv2Settlement", () => {
       const { [sellToken]: sellPrice, [buyToken]: buyPrice } = prices;
       const computeExecutedTradeForOrderVariant = async (
         { kind, partiallyFillable }: OrderFlags,
-        executedAmount?: BigNumber,
+        tradeExecution?: Partial<TradeExecution>,
```

Comment:
Note: this is what was called `trade` in other functions above and I proposed to change. Also this is a good name.